attestation-agent/attestation-agent/src/lib.rs

```diff
@@ -70,21 +75,54 @@ pub trait AttestationAPIs {
 /// Attestation agent to provide attestation service.
 pub struct AttestationAgent {
     _config: Config,
+    eventlog: EventLog,
 }
 
 impl Default for AttestationAgent {
     fn default() -> Self {
         if let Ok(_config) = Config::try_from(config::DEFAULT_AA_CONFIG_PATH) {
-            return AttestationAgent { _config };
+            let eventlog = EventLog::new().expect("eventlog initialize");
```

Comment:
is the idea to make AA to always log independent of the underlying extension mechanism used by the TEE?

Reply:
Yes. AA will do the conversion logic from events into underlying TEE. After kernel has got a converged eventlog and rtmr API. AA could call that API to extend, and also wrap the events into the target log format.